src/app/tap/features/RecordRhythm/components/RhythmBar/style.module.scss

```
@@ -0,0 +1,32 @@
+.container {
+	border-radius: 4px;
+	// overflow: hidden;
+}
+  
+.progressContainer {
+	width: 100%;
+	background-color: #eee;
```

統一するために色は直接指定せずにSCSS変数を使って欲しいです！
```suggestion
	background-color: $background-color
```

数字と色は全てSCSS変数を使うことが**理想**(結構面倒なので普段はやらない)です
`border-radius` 等の`src/styles/*.module.scss` で定義されているものは他のも置き換えて欲しいです

---

ゲージが溜まってない時の灰色の部分なんだけど`src/styles/*.module.scss`の方にない色なんだけど勝手に増やしていいってこと?

---

Done